src/docfx/SubCommands/BuildCommand.cs

```diff
@@ -307,45 +301,30 @@ private static void MergeGitContributeToConfig(BuildJsonConfig config)
             }
         }
 
-        internal static Dictionary<string, FileMetadataPairs> GetFileMetadataFromOption(ListWithStringFallback fileMetadataFilePaths, Dictionary<string, FileMetadataPairs> fileMetadataFromConfig)
```

Comment:
we support `--globalMetadata` in build command option but not support `--fileMetadata`, so there's something different. I think it would make the code too complicated if merge this two method into one?

---
In reply to: [73815535](https://github.com/dotnet/docfx/pull/575#discussion_r73815535) [](ancestors = 73815535)